adding component structure chapter (#54)
infinum/eightshift-docs: website/docs/basics/blocks-intro.md

### Do I need to have storybook stories in my block?

> No you don't. But we provided you an ability to use storybook for all of your blocks and components. Why not use it.

Reviewer: ```suggestion
No, you don't. But we provided you with the ability to use the storybook for all of your blocks and components. Why not use it? It will speed up your development time. Trust us 🙂.
```

### What is a wrapper?

> All details about wrapper you can find in [this chapter](blocks-wrapper).

Reviewer: ```suggestion
All the details about wrapper can be found in [this chapter](blocks-wrapper).
```

### Do you support inner blocks?

> Yes, we support everything that core natively supports. How to use them you can find in [this chapter](block-manifest).

Reviewer: ```suggestion
Yes, we support everything that core natively supports.  You can find more information about how to use them in [this chapter](block-manifest).
```

### Why do you use manifest.json in all blocks and components?

> So we can provide content, attributes, options and much more across multiple different files in multiple programing languages. More details read in [this chapter](block-manifest).

Reviewer: ```suggestion
So that we can provide content, attributes, options, and much more across multiple different files in multiple contexts (in both PHP and JS). You can read all about it in [this chapter](block-manifest).
```

### Why do you use global manifest.json

> Basicity the same answer as the previous one but in the global manifest.json we have options that are shared across all blocks and components. More details read in [this chapter](blocks-global-manifest).

Reviewer: ```suggestion
Basically the same answer as the previous one. In the global `manifest.json` we have options that are shared across all blocks and components. Read more about it in [this chapter](blocks-global-manifest).
```

> ### How do you use components in blocks?

Reviewer 2: ```suggestion
### How do I use components in blocks?
```

(all other questions are written from perspective of user so might as well this one)

> Yes we do, please referrer to [this chapter](blocks-component-in-block).

Reviewer: ```suggestion
Yes, we do. Please check [this chapter](blocks-component-in-block) out for more information.
```

> Yes we do, please referrer to [this chapter](blocks-component-in-block).

Reviewer 2: ```suggestion
Please referrer to [this chapter](blocks-component-in-block).
```

### How do I use multiple heading components in my block?

> Please referrer to [this chapter](blocks-component-in-block).

Reviewer: ```suggestion
You can follow the instructions given in [this chapter](blocks-component-in-block).
```
